src/components/jury-description.jsx

```diff
@@ -7,8 +8,18 @@ import { Row, Col } from "react-bootstrap";
 export default React.createClass({
   displayName: "JuryDescription",
   render() {
-    return <Row className="white-section">
-      <div className="jury-icon"><span className={this.props.iconClass}></span></div>
+    let juryIcon = null;
+    if (this.props.iconClass) {
+      let juryIconClass = ctx("section-icon", this.props.iconClass);
+      juryIcon = (
```

Comment:
Am observat ca oamenii folosesc in general prin cod de React ca se foloseste asa cu paranteze.